src/components/application_manager/test/commands/mobile/system_request_test.cc

```diff
@@ -68,23 +71,35 @@ const std::string kAppPolicyId = "fake-app-id";
 const uint32_t kDefaultTimeout = 1000u;
 const std::string kAppFolderName = "fake-app-name";
 const std::string kAppStorageFolder = "fake-storage";
-const std::string kSystemFilesPath = "/fake/system/files";
+const std::string kSystemFilesPath = "fake/system/files";
 const std::string kFileName = "Filename";
+const mobile_apis::RequestType::eType kRequestHttpType =
+    mobile_apis::RequestType::HTTP;
+const mobile_apis::RequestType::eType kRequestClimate =
+    mobile_apis::RequestType::CLIMATE;
+const mobile_apis::RequestType::eType kRequestQueryApps =
+    mobile_apis::RequestType::QUERY_APPS;
 }  // namespace
 
 class SystemRequestTest
     : public CommandRequestTest<CommandsTestMocks::kIsNice> {
  public:
-  SystemRequestTest() : mock_app_(CreateMockApp()) {}
+  SystemRequestTest()
+      : mock_app_(CreateMockApp())
+      , binary_data_folder(kAppStorageFolder + "/" + kAppFolderName + "/")
+      , app_full_file_path(binary_data_folder + kFileName)
+      , binary_data_(10, 10) {}
```

Comment:
@BSolonenko is is very not obvious that  binary_data_ will be filled with 10 values of 10 .
Please use fill value something like 0xFF.